client/src/productOverview/rating.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import StarsRating from 'stars-rating';
+
+
+class Rating extends React.Component {
+  constructor(props) {
+    super(props);
+
+  }
+
+  getRate() {
+
+  }
+
+  render() {
+    return (
+      <div>★★★★★
+      </div>
+
+    );
+  }
+}
+
+export default Rating;
```

Comment:
Looks good! Pro tip: You can define your class as below and skip the export at the bottom if you prefer.

`export default class Rating extends React.Component`